.github/workflows/ci.yaml

```diff
@@ -146,6 +142,17 @@ jobs:
             -w compute_xsmall \
             -d METADATA
 
+      - name: policies
+        shell: bash
+        run: |
+          schemachange \
+            -f admin/policies \
+            -a $SNOWFLAKE_ACCOUNT \
+            -u $SNOWFLAKE_USER \
+            -r ACCOUNTADMIN \
+            -w compute_xsmall \
+            -d METADATA
```

Comment:
> immediately call USE SCHEMA POLICY_DB.PUBLIC in the scripts themselves so that our policy objects are referenced in their usual place.

Can you specify the schema as an argument in the schemachange call instead? I looked in the docs and didn't find anything, but the docs seem to be outdated because they don't describe the database argument `-d` either.

Reply:
The `-d` argument doesn't necessarily specify the database in which the scripts execute, but it does specify the database in which schemachange writes to the CHANGE_HISTORY table. We want to use the CHANGE_HISTORY table in the METADATA database since we anticipate aggregating all of our admin-level database objects in a single database as part of [SNOW-240](https://sagebionetworks.jira.com/browse/SNOW-240).

btw, if you install schemachange locally you can call `schemachange --help` to see all possible arguments.

Reply:
I see - and configuring the schema wouldn't work in this case because `PUBLIC` is under a different db. Thanks!